### PR TITLE
add unit test for pkg/apis/core/toleration.go

### LINKED ISSUE #56753: add testcase toleration.go function
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind  bug

> Uncomment only one, leave it on its own line: 
>
> /kind bug
> /kind feature

**What happened**:

**What you expected to happen**:
add testcase  for toleration.go function.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
improve coverage for function

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes [https://github.com/kubernetes/kubernetes/issues/56753](https://github.com/kubernetes/kubernetes/issues/56753)

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by gyliu513 — COMMENTED** [review]
/lgtm

**Comment by stewart-yu** [comment]
@ gyliu513

**Review by resouer — CHANGES_REQUESTED** [review]
There's a build error in this patch, please fix it. :)

**Comment by stewart-yu** [comment]
Thanks, fit it @resouer .Someone had move origin go file package.

**Review by resouer — APPROVED** [review]

File: pkg/apis/core/toleration_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import "testing"
+
+func TestMatchToleration(t *testing.T) {
```
Comment (bsalamat) [inline pkg/apis/core/toleration_test.go]: `MatchToleration()` checks that all fields of `Toleration` are the same, except `TolerationSeconds`. It ignores `TolerationSeconds`. Please add a case to verify that `MatchToleration()` returns true even when `TolerationSeconds` of two `Toleration`s are not equal.

Comment (stewart-yu) [reply pkg/apis/core/toleration_test.go]: Done, thanks @bsalamat

**Comment by bsalamat** [comment]
/lgtm

Thanks, @stewart-yu!

**Comment by bsalamat** [comment]
cc/ @kubernetes/api-approvers

**Comment by stewart-yu** [comment]
cc @lavalamp

**Pull request merged**